### PR TITLE
Release notes 2.10

### PR DESCRIPTION
**Pull request by KodiaqQ**
### Changes

- Added v2.10.0 template;

### Reason for changes

- Upcoming release;

### Related tickets

- 138131;

#### For the contributors:

1. Please add your changes (as the commit to the branch) to the list according to the template and previous notes;
2. Do not add tests-related notes;
3. Provide the list of the PRs (for all your notes) in the comment for the discussion;

### CODE REVIEW

**Comment by KodiaqQ** [comment]
@alexsu52, @ljaljushkin, @kshpv,  @daniil-lyakhov,  @ljaljushkin, @andreyanufr, @andrey-churkin, @nikita-savelyevv, @l-bat, please update Release Notes with your PR-s according to the description.

**Comment by daniil-lyakhov** [comment]
(Torch) `nncf.quantize` funciton could be used as quantization initialization for Quantization-aware training: https://github.com/openvinotoolkit/nncf/pull/2404
Quantization-aware training of Resnet18 Model

**Review by nikita-savelyevv — APPROVED** [review]

File: ReleaseNotes.md [hunk 1]
```diff
@@ -1,5 +1,54 @@
 # Release Notes
 
+## New in Release 2.10.0
+
+Post-training Quantization:
+
+- Features:
+  - Introduced the subgraph defining functionality for the `nncf.IgnoredScope()` option.
+  - Introduced batch support for the optimization process via `nncf.*` methods. This also includes the new option `batchwise_statistics` for the `nncf.AdvancedQuantizationParameters()`.
```
Comment (MaximProshin) [inline ReleaseNotes.md:9]: I think it will be clearer to say "batch size > 1".

File: ReleaseNotes.md [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
+- Fixes:
+  - Fixed issue with the `nncf.OverflowFix` parameter absence in some scenarios.
+  - Aligned the list of correctable layers for the FastBiasCorrection algorithm between PyTorch, OpenVINO and ONNX backends.
+  - Fixed Mobilenetv2 ([PyTorch](examples/post_training_quantization/torch/mobilenet_v2), [ONNX](examples/post_training_quantization/onnx/mobilenet_v2), [OpenVINO](examples/post_training_quantization/openvino/mobilenet_v2)) examples for the Windows platform.
+  - (OpenVINO) Fixed [an Anomaly Classification](examples/post_training_quantization/openvino/anomaly_stfpm_quantize_with_accuracy_control) for the Windows platform.
```
Comment (MaximProshin) [inline ReleaseNotes.md:14]: an Anomaly Classification => Anomaly Classification example

**Review by alexsu52 — APPROVED** [review]
There are my PRs in the release notes. Thanks.

File: ReleaseNotes.md [hunk 3]
```diff
@@ -1,5 +1,54 @@
...
+- Fixes:
+  - (PyTorch) Fixed `apply_args_defaults` function issue.
+- Improvements:
+  - (PyTorch) Updated command creation behaviour to reduce the number of adapters.
+- Deprecations/Removals:
+  - (PyTorch) Removed the `binarization` algorithm.
+  - NNCF installation via `pip install nncf[<framework>]` option is now deprecated.
+- Requirements:
+  - Updated PyTorch (2.2.1) and CUDA (12.1) versions.
+  - Updated  ONNX version (1.16.0).
```
Comment (MaximProshin) [inline ReleaseNotes.md:50]: ORT was bumped as well.

**Review by daniil-lyakhov — APPROVED** [review]

File: ReleaseNotes.md [hunk 4]
```diff
@@ -1,5 +1,54 @@
...
+  - [Post-Training Optimization of LLaVA Next Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/llava-next-multimodal-chatbot/llava-next-multimodal-chatbot.ipynb)
+  - [Post-Training Optimization of AnimateAnyone Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/animate-anyone/animate-anyone.ipynb)
+  - [Post-Training Optimization of YOLOv8-OBB Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/yolov8-optimization/yolov8-obb.ipynb)
+  - [Post-Training Optimization of LLM Agent](https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/llm-agent-langchain/llm-agent-langchain.ipynb)
+
+Compression-aware training:
+
+- Features:
+  - (PyTorch) `nncf.quantize` method now may be used as quantization initialization for Quantization-Aware Training.
+  - (PyTorch) Added a [Resnet18-based example](examples/quantization_aware_training/torch/resnet18) with the transition from the Post-Training Quantization to a Quantization-Aware Training algorithm.
```
Comment (MaximProshin) [inline ReleaseNotes.md:40]: I would combine it with the previous bullet.

**Review by MaximProshin — APPROVED** [review]

**Pull request merged**